docs/operator-guides/collecting-metrics.md

```diff
@@ -145,17 +153,33 @@ The following metrics are available. They're published every minute.
       <td>1.5.0</td>
     </tr>
     <tr>
-      <td rowspan="3"><code>airbyte.api_requests</code></td>
-      <td><code>workspace_id</code></td>
-      <td>bed3b473-1518-4461-a37f-730ea3d3a848</td>
+      <td><code>service</code></td>
+      <td>airbyte-worker</td>
     </tr>
     <tr>
-      <td><code>endpoint</code></td>
-      <td>/v1/connections/sync</td>
+      <td rowspan="7"><code>airbyte.api_requests</code></td>
+      <td><code>uri</code></td>
+      <td>https://api.airbyte.com/v1/applications</td>
```

Comment:
@ian-at-airbyte This is still just the bit without the host, e.g. `/v1/application/create`